starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaDrmBridge.java

```diff
@@ -772,7 +772,7 @@ private void getConnectedHdcpLevelInfoV29(MediaDrm mediaDrm) {
         Log.i(TAG, "MediaDrm HDCP Level is HDCP_LEVEL_UNKNOWN.");
         break;
       default:
-        Log.i(TAG, String.format("Unknown MediaDrm HDCP level %d.", hdcpLevel));
+        Log.i(TAG, String.format(Locale.US, "Unknown MediaDrm HDCP level %d.", hdcpLevel));
```

Comment:
Should this one import `java.util.Locale`?

Reply:
Fixed above.

Reply:
It does